pkg/apis/pingcap/v1alpha1/tidb_config_wraper.go

```diff
@@ -26,12 +26,14 @@ import (
 var _ stdjson.Marshaler = &TiDBConfigWraper{}
 var _ stdjson.Unmarshaler = &TiDBConfigWraper{}
 
+// NewTiDBConfig returns an empty config structure
```

Comment:
```suggestion
```

Reply:
exported function should have at least a simple comment, so this should be kept.